[2025-03-09T17:43:49.864+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-09T17:43:49.998+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_nyc_fire_incidents_dag.extract_data_task manual__2025-03-09T17:43:47.604468+00:00 [queued]>
[2025-03-09T17:43:50.004+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_nyc_fire_incidents_dag.extract_data_task manual__2025-03-09T17:43:47.604468+00:00 [queued]>
[2025-03-09T17:43:50.005+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-09T17:43:50.019+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_data_task> on 2025-03-09 17:43:47.604468+00:00
[2025-03-09T17:43:50.026+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=89) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-09T17:43:50.027+0000] {standard_task_runner.py:72} INFO - Started process 91 to run task
[2025-03-09T17:43:50.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_nyc_fire_incidents_dag', 'extract_data_task', 'manual__2025-03-09T17:43:47.604468+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/fire_incidents_dag.py', '--cfg-path', '/tmp/tmp5p05dkz6']
[2025-03-09T17:43:50.028+0000] {standard_task_runner.py:105} INFO - Job 580: Subtask extract_data_task
[2025-03-09T17:43:50.069+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_nyc_fire_incidents_dag.extract_data_task manual__2025-03-09T17:43:47.604468+00:00 [running]> on host fad361e322ea
[2025-03-09T17:43:50.153+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_nyc_fire_incidents_dag' AIRFLOW_CTX_TASK_ID='extract_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T17:43:47.604468+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T17:43:47.604468+00:00'
[2025-03-09T17:43:50.154+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-09T17:43:50.154+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-09T17:43:50.155+0000] {logging_mixin.py:190} INFO - Current task name:extract_data_task state:running start_date:2025-03-09 17:43:49.998906+00:00
[2025-03-09T17:43:50.155+0000] {logging_mixin.py:190} INFO - Dag name:etl_nyc_fire_incidents_dag and current dag run status:running
[2025-03-09T17:43:50.155+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-09T17:43:50.156+0000] {logging_mixin.py:190} INFO - Extracting NYC Fire Incidents Data via API....
[2025-03-09T17:44:05.621+0000] {logging_mixin.py:190} INFO - Connected to API...
[2025-03-09T17:44:05.657+0000] {logging_mixin.py:190} INFO - Extracting is Complete
[2025-03-09T17:44:09.724+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-09T17:44:09.737+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-09T17:44:09.739+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_nyc_fire_incidents_dag, task_id=extract_data_task, run_id=manual__2025-03-09T17:43:47.604468+00:00, execution_date=20250309T174347, start_date=20250309T174349, end_date=20250309T174409
[2025-03-09T17:44:09.771+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-09T17:44:09.771+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-09T17:44:09.772+0000] {logging_mixin.py:190} INFO - Dag name:etl_nyc_fire_incidents_dag queued_at:2025-03-09 17:43:47.614688+00:00
[2025-03-09T17:44:09.772+0000] {logging_mixin.py:190} INFO - Task hostname:fad361e322ea operator:PythonOperator
[2025-03-09T17:44:09.807+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-09T17:44:09.833+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-09T17:44:09.835+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
